### PR TITLE
Changelog for #18670

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

File: doc/sphinx/changes.rst [hunk 1]
```diff
@@ -654,6 +654,14 @@ Kernel
   (`#18599 <https://github.com/coq/coq/pull/18599>`_,
   by Guillaume Melquiond).
 
+Notations
+^^^^^^^^^
+
+- **Changed:**
+  :warn:`Found no matching notation to enable or disable.` is a warning instead of an error
+  (`#18670 <https://github.com/coq/coq/pull/18670>`_,
+  by Pierre Roux).
```
Comment (SkySkimmer) [inline doc/sphinx/changes.rst:663]: cc @proux01

Comment (proux01) [reply doc/sphinx/changes.rst:663]: I could maybe have written "is now" rather than "is", but that's fine.

**Review by proux01 — APPROVED** [review]

**Comment by proux01** [comment]
But apparently this now breaks compilation of the refman? how comes? wasn't it already included in the unreleased changes section?

**Comment by SkySkimmer** [comment]
No, this is all new text

**Comment by proux01** [comment]
Indeed, I forgot the changelog there, my bad 😢

**Comment by SkySkimmer** [comment]
should work now

**Comment by proux01** [comment]
Indeed, thanks for fixing (once again) my mistakes.

@coqbot merge now

**Comment by proux01** [comment]
@coqbot merge now

**Pull request merged**